.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Build
 on:
   push:
     branches:
-      - master
+      - main
     tags:
       - v*
```

Comment:
Is this really required? Can you push a tag that doesn't also push to the main branch?

Reply:
I don't know enough about github actions, but @jreese probably does

Reply:
Technically, yes, you can create a tag, and `git push --tags` or `git push origin <tagname>` to send the tag without updating main. There's also nothing in git that requires tags to point at commits reachable from named branches.